### PR TITLE
Format python snippets

### PR DESCRIPTION
**Pull request by rnarkk**

### CODE REVIEW

File: content/en/docs/languages/python/basics.md [hunk 1]
```diff
@@ -220,16 +220,18 @@ that sends multiple `Feature`s to the client.
...
+    right = max(request.lo.longitude, request.hi.longitude)
+    top = max(request.lo.latitude, request.hi.latitude)
+    bottom = min(request.lo.latitude, request.hi.latitude)
+    for feature in self.db:
+        if (
+            feature.location.longitude >= left and
+            feature.location.longitude <= right and
+            feature.location.latitude >= bottom and
+            feature.location.latitude <= top
+        ):
```
Comment (sergiitk) [inline content/en/docs/languages/python/basics.md:233]:
```suggestion
        if (
            feature.location.longitude >= left
            and feature.location.longitude <= right
            and feature.location.latitude >= bottom
            and feature.location.latitude <= top
        ):
```

File: content/en/docs/languages/python/basics.md [hunk 2]
```diff
@@ -244,25 +246,25 @@ request values and returns a single response value.
...
+            feature_count += 1
+        if prev_point:
+            distance += get_distance(prev_point, point)
+        prev_point = point
+
+    elapsed_time = time.time() - start_time
+    return route_guide_pb2.RouteSummary(point_count=point_count,
+                                        feature_count=feature_count,
+                                        distance=int(distance),
+                                        elapsed_time=int(elapsed_time))
```
Comment (sergiitk) [inline content/en/docs/languages/python/basics.md:267]:
```suggestion
    return route_guide_pb2.RouteSummary(
        point_count=point_count,
        feature_count=feature_count,
        distance=int(distance),
        elapsed_time=int(elapsed_time),
    )
```

File: content/en/docs/languages/python/basics.md [hunk 3]
```diff
@@ -290,12 +292,12 @@ start up a gRPC server so that clients can actually use your service:
...
-  server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
-  route_guide_pb2_grpc.add_RouteGuideServicer_to_server(
-      RouteGuideServicer(), server)
-  server.add_insecure_port('[::]:50051')
-  server.start()
-  server.wait_for_termination()
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    route_guide_pb2_grpc.add_RouteGuideServicer_to_server(
+        RouteGuideServicer(), server)
+    server.add_insecure_port('[::]:50051')
```
Comment (sergiitk) [inline content/en/docs/languages/python/basics.md:298]:
```suggestion
    route_guide_pb2_grpc.add_RouteGuideServicer_to_server(RouteGuideServicer(), server)
    server.add_insecure_port("[::]:50051")
```

File: content/en/docs/languages/python/quickstart.md [hunk 4]
```diff
@@ -171,11 +171,11 @@ this:
 ```py
 class Greeter(helloworld_pb2_grpc.GreeterServicer):
 
-  def SayHello(self, request, context):
-    return helloworld_pb2.HelloReply(message=f'Hello, {request.name}!')
+    def SayHello(self, request, context):
+        return helloworld_pb2.HelloReply(message=f'Hello, {request.name}!')
```
Comment (sergiitk) [inline content/en/docs/languages/python/quickstart.md:175]:
```suggestion
    def SayHello(self, request, context):
        return helloworld_pb2.HelloReply(message=f"Hello, {request.name}!")
```

File: content/en/docs/languages/python/quickstart.md [hunk 5]
```diff
@@ -171,11 +171,11 @@ this:
...
 
-  def SayHelloAgain(self, request, context):
-    return helloworld_pb2.HelloReply(message=f'Hello again, {request.name}!')
+    def SayHelloAgain(self, request, context):
+        return helloworld_pb2.HelloReply(message=f'Hello again, {request.name}!')
```
Comment (sergiitk) [inline content/en/docs/languages/python/quickstart.md:178]:
```suggestion
    def SayHelloAgain(self, request, context):
        return helloworld_pb2.HelloReply(message=f"Hello again, {request.name}!")
```

**Review by sergiitk — COMMENTED** [review]
Thanks for the contribution!

We've switched to [black](https://black.readthedocs.io/en/stable/index.html) formatting, so I left some additional comments to be compliant.

**Review by sergiitk — APPROVED** [review]

**Review by ejona86 — APPROVED** [review]

**Comment by ejona86** [comment]
Thank you!

**Pull request merged**